### DISCUSSION TITLE
Unable to use as cronjob

### DISCUSSION THREAD

**Discussion by MqlBql** [UnconnectedBedna/shrink-backup#32] [Q&A]
Edit : Posted an issue instead.

**Comment by UnconnectedBedna** [comment]
#33 

I never heard back from you buddy, but the failing second backup you had WAS a problem that started showing up suddenly.
The UUID was missing in the `lsblk` command, so after a lot of head scratching and hair pulling I figured I just do a loop 3 times to give the system time and that seems to have fixed it.

I actually missed this post in the first place and wanted to clarify in case someone was wondering why I was referencing commits to an "invalid" bug report. :slightly_smiling_face:

**Reply by MqlBql** [reply]
Hello.
So sorry for the late answer. Very rude of me. Have been pretty busy at work lately. I will test your fix this week end and let you know how it goes.
However, don't bother too much with this as it works very well with the systemd method you provided me.

**Reply by UnconnectedBedna** [reply]
Should work with both, the reason it failed has been secured with the loop.
Might I guess you store your img on either a slow drive or over network? xD

No worries, your issue made me test it on dietPi so I could add that to supported distros. :)

But if it still fails with using cron (just please make sure you run it as sudo/root and that it is not hanging on a passwd check or smthn) please let me know!! I WANT it to work with it, and it should...
